src/phoenix/trace/span_evaluations.py

```diff
@@ -152,6 +157,24 @@ def __init_subclass__(
             tuple(sorted(prod)) for prod in product(*cls.index_names.keys())
         )
 
+    def to_parquet(self, path: Union[str, Path]) -> None:
+        table = Table.from_pandas(self.dataframe)
+        table = table.replace_schema_metadata(
+            {
+                **table.schema.metadata,
+                b"openinference": json.dumps({"eval_name": self.eval_name}).encode("utf-8"),
```

Comment:
I'm not 100% sure this falls under openinference but for a lack of a better namespace, this could work

Reply:
Renamed to "arize"